### PR TITLE
Refactor exports

### PR DESCRIPTION
**Pull request by Page-**

### CODE REVIEW

File: src/sbvr-api/errors.ts [hunk 1]
```diff
@@ -8,10 +10,7 @@ export class SbvrValidationError extends TypedError {}
 
 export class TranslationError extends TypedError {}
 export class ParsingError extends TypedError {}
-
-export interface AnyObject {
-	[index: string]: any;
-}
+export { AnyObject };
```
Comment (dfunckt) [inline src/sbvr-api/errors.ts:13]: I think it's best to not export this one, getting it from common-types everywhere is better IMO.

File: src/sbvr-api/sbvr-utils.ts [hunk 2]
```diff
@@ -59,7 +59,6 @@ import {
 	statusCodeToError,
 	TranslationError,
 	UnauthorizedError,
-	AnyObject,
```
Comment (dfunckt) [inline src/sbvr-api/sbvr-utils.ts:62]: Github won't let me comment there, but it seems we're re-exporting `./permissions` on line 46 above. Can that be removed?

File: src/sbvr-api/errors.ts (hunk 1 above)
Comment (Page-) [reply src/sbvr-api/errors.ts:13]: This is necessary for backwards compatibility, I'll drop this in the major

File: src/sbvr-api/sbvr-utils.ts (hunk 2 above)
Comment (Page-) [reply src/sbvr-api/sbvr-utils.ts:62]: It's also a major change, but I do plan to

**Pull request merged**